opa/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    'name': "Open Academy",
+    'summary': """
+        Manage your academy.""",
+    'author': "Vauxoo, Antonio Aguilar <antonio@vauxoo.com>",
+    'website': "https://vauxoo.com",
+    'category': 'Services',
+    'version': '0.1',
```

Comment:
Change the version to the actual odoo version

```suggestion
    'version': '15.0.1.0.1',
```

Reply:
Done